py/desisim/batch/pixsim.py

```diff
@@ -32,14 +33,17 @@
 log = get_logger()
 
 def batch_newexp(batchfile, flavors, nspec=5000, night=None, expids=None,
-    nodes=None, pixprod=None, desi_spectro_sim=None, tileids=None):
+    nodes=None, pixprod=None, desi_spectro_sim=None, tileids=None, seed=None):
     '''
     Write a slurm batch script for run newexp-desi for the list of flavors
     '''
     nexp = len(flavors)
     timestr = '00:30:00'
     logfile = '{}.%j.log'.format(batchfile)
-    
+
+    np.random.seed(seed)
```

Comment:
Yes, in fact `None` is the default value if you call `numpy.random.seed()` without an argument.  This is passed to `numpy.random.RandomState` which is more explicit that `None` is valid:

>    If `seed` is ``None``, then `RandomState` will try to read data from
    ``/dev/urandom`` (or the Windows analogue) if available or seed from
    the clock otherwise.